domains/certificates/Query.API/API.UnitTests/MeasurementsSyncer/SlidingWindowServiceTest.cs

```diff
@@ -19,10 +21,12 @@ public class SlidingWindowServiceTest
     private readonly Gsrn _gsrn = new Gsrn(GsrnHelper.GenerateRandom());
     private readonly UnixTimestamp _now = UnixTimestamp.Now();
     private readonly IMeasurementSyncMetrics _measurementSyncMetrics = Substitute.For<IMeasurementSyncMetrics>();
+    private readonly MeasurementsSyncOptions _options = new();
+
 
     public SlidingWindowServiceTest()
     {
-        _sut = new SlidingWindowService(_measurementSyncMetrics);
+        _sut = new SlidingWindowService(_measurementSyncMetrics, Options.Create(_options));
```

Comment:
I would be afraid of having side effects overwritting options in each test. I would prefer newing up SlidingWindowService in each test with it's own options object, since the options is not the same.

Reply:
I could set it to a default value of 0 in the class, and then modify at will in each relevant test instead?

Reply:
There you go: https://github.com/Energinet-DataHub/energy-origin/pull/1575/commits/0b2b0684a3473877ee54ae95ece1879fada158b3

No more side effects ;)

Reply:
I've read up on how xUnit works under the hood. It is implicitly newing up the service for each test run, and runs tests sequentially. There is therefore no side effects that can occur:

https://xunit.net/docs/shared-context.html